config/default/kustomization.yaml

```diff
@@ -1,26 +1,10 @@
-# Adds namespace to all resources.
```

Comment:
Much of this commented code came from the aws-load-balancer-controller. I've tried to remove what we don't need. One notable difference is that the gateway controller enables the webhook configuration by default, even though you still have to opt into it for any given namespace.